index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+
+      <div id="title">
+        Dr. Norman Borlaug
+        <div id="tribute-info">The man who saved a billion lives</div>
+      </div>
+
+
+      <div id="img-div">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="" id="image">
+        <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+      </div>
```

Comment:
Please use `<section>` tag to divide the content of the document into separate sematic blocks. Have a look at the definition in this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)